doc/maintaining/cli.rst

```diff
@@ -0,0 +1,572 @@
+.. _cli:
+
+======================
+Command Line Interface (CLI)
+======================
+
+NB: From CKAN 2.9 onwards the CKAN configuration file is named 'ckan.ini'. Previous names: 'production.ini', 'development.ini' plus others may also still appear in documentation and the software. These legacy names will eventually be phased out
+
+Most common CKAN administration tasks can be carried out from the command line
+on the server that CKAN is installed on, using the ``ckan`` command.
+
+If you have trouble running ckan commands, see
+`Troubleshooting ckan Commands`_ below.
+
+.. note::
+
+   Before running a CKAN ``ckan`` command, you have to activate your CKAN
+   virtualenv and change to the ``ckan``  directory, for example:
+
+   .. parsed-literal::
+
+      |activate|
+      cd |virtualenv|/src/ckan
+
+   To run a ckan command without activating the virtualenv first, you have
+   to give the full path the ckan script within the virtualenv, for example:
+
+   .. parsed-literal::
+
+      |virtualenv|/bin/ckan -c |/path/to/ckan.ini| --plugin=ckan user list 
+
+   To run a ckan command without changing to the ckan directory first, add
```

Comment:
As mentioned above, `ckan` doesn't use `--plugin` and available globally with active virtual environment, so the whole paragraph can be dropped